src/process_utility.c

```diff
@@ -314,6 +320,94 @@ process_drop(Node *parsetree)
 	return handled;
 }
 
+typedef struct DropTriggerCtx
```

Comment:
You don't need this if only passing a single value/pointer. Just pass the `trigger_name` directly instead.